src/core_atmosphere/physics/mpas_atmphys_manager.F

```diff
@@ -548,10 +549,10 @@ subroutine physics_run_init(configs,mesh,state,clock,stream_manager)
 
 !set alarm for updating the background surface albedo and the greeness fraction:
  call mpas_set_timeInterval(alarmTimeStep,timeString=config_greeness_update,ierr=ierr)
- alarmStartTime = startTime
+    alarmStartTime = startTime
  call mpas_add_clock_alarm(clock,greenAlarmID,alarmStartTime,alarmTimeStep,ierr=ierr)
     if(ierr /= 0) &
-       call physics_error_fatal('subroutine physics_init: error creating alarm greeness')
+ call physics_error_fatal('subroutine physics_init: error creating alarm greeness')
```

Comment:
Change in spacing

Reply:
@AndersJensen-NOAA Line 554 is restored.